test_settings.py

```diff
@@ -0,0 +1,43 @@
+"""
+These settings are here to use during tests, because django requires them.
+
+In a real-world use case, apps in this project are installed into other
+Django applications, so these settings will not be used.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+from os.path import abspath, dirname, join
+
+
+def root(*args):
```

Comment:
Nit: Is this method used?

Reply:
It's not, but it is part of the cookiecutter template, so seemed like a standardized way to handle that functionality and worth keeping.